### PR TITLE
run formatter over pre-prod services

### PR DESCRIPTION
**Pull request by martingartonft**

### CODE REVIEW

File: services.yaml [hunk 1]
```diff
@@ -1,135 +1,132 @@
...
+- name: coco-kafka-bridge@.service
+  version: v6
+  count: 2
+- name: content-ingester-sidekick@.service
+  count: 2
+- name: content-ingester@.service
+  version: v33
+  count: 2
+- name: diamond.service
+- name: document-store-api-sidekick@.service
```
Comment (Zapadlo) [inline services.yaml]: Dup entry

File: services.yaml [hunk 2]
```diff
@@ -1,135 +1,132 @@
...
+  count: 2
+- name: document-store-api-sidekick@.service
+  count: 2
+- name: document-store-api@.service
+  version: v31
+  count: 2
+- name: document-store-api@.service
```
Comment (Zapadlo) [inline services.yaml]: Same

File: services.yaml [hunk 3]
```diff
@@ -1,135 +1,132 @@
...
+- name: methode-list-transformer-sidekick@.service
+  count: 2
+- name: methode-list-transformer@.service
+  version: v9
+  count: 2
+- name: methodeapi-endpoint.service
+- name: mongo-backup.service
+  desiredState: loaded
+- name: mongo-backup.timer
+- name: mongodb-all.service
```
Comment (Zapadlo) [inline services.yaml]: Dupe

**Comment by martingartonft** [comment]
So helping find dupes if showing the value of this already.
I'm not trying to de-dupe with this PR though.
(partly because I don't know what the "right" versions of some things are)
seem ok?

**Comment by Zapadlo** [comment]
Yep +1

**Pull request closed without merging**